### PR TITLE
feat(react): #MAG-391 display magneto elements

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
image

## Checklist tests
check display and card interactions (tooltips)

## Issue ticket number and link
[MAG-391](https://jira.support-ent.fr/browse/MAG-391)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/board-list/BoardList.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import { Icon } from "@mdi/react";
+import { useTranslation } from "react-i18next";
+
+
+
+
+export const BoardList = () => {
+    const { user, currentApp } = useOdeClient();
+    const { t } = useTranslation();
+    // const [isToasterOpen, setIsToasterOpen] = useToaster();
```
Comment (flmartineau) [inline frontend/src/components/board-list/BoardList.tsx:25]: Tu laisse le commentaire ?

File: frontend/src/components/board-list/BoardList.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        isPublic: false,
+        isShared: true,
+        isDeleted: false,
+        sortBy: 'modificationDate',
+        page: 0
+    }) || {};
+  
+    let boardData;
+    if (getBoardsError) {
+      console.log("error");
```
Comment (flmartineau) [inline frontend/src/components/board-list/BoardList.tsx:39]: Enlève les console.log

File: frontend/src/components/board-list/BoardList.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                        ...springs
+                        }}
+                    >
+                        <Card
+                            app={currentApp!}
+                            options={{
+                                type: "board",
+                                title,
+                            }}
+                            // onClick={() => {setIsToasterOpen()}}
```
Comment (flmartineau) [inline frontend/src/components/board-list/BoardList.tsx:78]: commentaire en trop ?

File: frontend/src/components/board-list/BoardList.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+    const { data: myBoardsResult, isLoading: getBoardsLoading, error: getBoardsError } = useGetBoardsQuery({
+        isPublic: false,
+        isShared: true,
+        isDeleted: false,
+        sortBy: 'modificationDate',
+        page: 0
+    }) || {};
+  
+    let boardData;
```
Comment (flmartineau) [inline frontend/src/components/board-list/BoardList.tsx:37]: possible de typer ?

**Comment by flmartineau** [comment]
Je vais pas faire plus de commentaires par rapports aux code commenté, si c'est normal, peut-être préciser dans la PR dans ce cas là ! Sinon c'est ok 😊

File: frontend/src/components/board-list/BoardList.tsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+
+export const BoardList = () => {
+    const { user, currentApp } = useOdeClient();
+    const { t } = useTranslation();
+    // const [isToasterOpen, setIsToasterOpen] = useToaster();
+
+    const userId = user ? user?.userId : "";
+
+    const { data: myBoardsResult, isLoading: getBoardsLoading, error: getBoardsError } = useGetBoardsQuery({
```
Comment (FlorentMr) [inline frontend/src/components/board-list/BoardList.tsx:29]: Aucun risque de synchronisation de données sans englober de useEffect?

File: frontend/src/components/create-tab/createTab.tsx [hunk 6]
```diff
@@ -63,8 +63,8 @@ export const CreateTab: FunctionComponent<props> = ({
     board.title = title;
     board.description = description;
     //TODO : change this to work with a future workspace file manager
-    board.imageUrl = await getUrl(thumbnail as File);
-    board.backgroundUrl = await getUrl(background as File);
+    // board.imageUrl = await getUrl(thumbnail as File);
```
Comment (FlorentMr) [inline frontend/src/components/create-tab/createTab.tsx:66]: Pourquoi commenter?

File: frontend/src/components/create-tab/createTab.tsx [hunk 7]
```diff
@@ -73,7 +73,7 @@ export const CreateTab: FunctionComponent<props> = ({
     board.displayNbFavorites = isFavoriteChecked;
     board.tags = tags;
 
-    createBoard(board.toJSON());
+    // createBoard(board.toJSON());
```
Comment (FlorentMr) [inline frontend/src/components/create-tab/createTab.tsx:76]: same

File: frontend/src/components/folder-list/FolderList.tsx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+    const { currentApp } = useOdeClient();
+    // const [isToasterOpen, setIsToasterOpen] = useToaster();
+
+    const { data: myFoldersResult, isLoading: getFoldersLoading, error: getFoldersError } = useGetFoldersQuery(false);
+    const { data: deletedFoldersResult, isLoading: getDeletedFoldersLoading, error: getDeletedFoldersError } = useGetFoldersQuery(true);
+
+    let folderData;
+    let deletedFolders;
+    if (getFoldersError || getDeletedFoldersError) {
+        console.log("error");
```
Comment (FlorentMr) [inline frontend/src/components/folder-list/FolderList.tsx:28]: logs

File: frontend/src/components/side-bar/SideBar.tsx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+  const { data: myFoldersResult, isLoading: getFoldersLoading, error: getFoldersError } = useGetFoldersQuery(false);
+  const { data: deletedFoldersResult, isLoading: getDeletedFoldersLoading, error: getDeletedFoldersError } = useGetFoldersQuery(true);
+
+  let myFoldersObject;
+  let deletedFoldersObject;
+  
+
+  if (getFoldersError || getDeletedFoldersError) {
+    console.log("error");
```
Comment (FlorentMr) [inline frontend/src/components/side-bar/SideBar.tsx:27]: logs

File: frontend/src/components/board-list/BoardList.tsx (hunk 1 above)
Comment (alicedraillard) [reply frontend/src/components/board-list/BoardList.tsx:25]: j'avais oublié de publier mes coms, c'est fait !

File: frontend/src/components/board-list/BoardList.tsx (hunk 5 above)
Comment (alicedraillard) [reply frontend/src/components/board-list/BoardList.tsx:29]: normalement non !

Comment (alicedraillard) [reply frontend/src/components/board-list/BoardList.tsx:29]: + ce sera géré dans le ticket suivant

**Review by FlorentMr — APPROVED** [review]

**Pull request merged**